### DISCUSSION TITLE
Personal owner

### DISCUSSION THREAD

**Discussion by AghaSKhan** [community/community#76640] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I logged in as agha.khan@surprisinggames.net and would like to add a repository, but it shows my personal owner's name as an owner. I want to create a new owner.

**Comment by lotirium** [comment]
If you want to add a repository on GitHub but it shows your personal name as the owner and you wish to create a new owner, you will need to create a new organization. Organizations in GitHub are shared accounts where groups of people can collaborate across many projects at once. Here’s how you can create a new organization:

1. Click your profile photo in the upper-right corner of any page, then click "Settings"
2. In the "Access" section of the sidebar, click "Organizations"
3. Next to the "Organizations" header, click "New organization"
4. Follow the prompts to create your organization​​

After setting up your organization, you can then create new repositories under this new entity, allowing for separation between your personal and organizational projects

**Reply by AghaSKhan** [reply]
Thank you.
